pkg/prove/querier.go

```diff
@@ -0,0 +1,56 @@
+package prove
+
+import (
+	"fmt"
+	"strconv"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	abci "github.com/tendermint/tendermint/abci/types"
+	"github.com/tendermint/tendermint/pkg/consts"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	"github.com/tendermint/tendermint/types"
+)
+
+const TxInclusionQueryPath = "txInclusionProof"
```

Comment:
[question] does this not need the `custom/` prefix that is used in https://github.com/celestiaorg/celestia-core/pull/859/files#diff-6572dddc535b4d2c98746d71dde97840af1328797e193db28376939a320be8f8R48 ?

Reply:
per comment in core, this one needs to not include the path as it will be removed/added by the cosmos-sdk